.web-docs/components/builder/iso/README.md

```diff
@@ -197,6 +197,9 @@ in the image's Cloud-Init settings for provisioning.
 - `cloud_init_storage_pool` (string) - Name of the Proxmox storage pool
   to store the Cloud-Init CDROM on. If not given, the storage pool of the boot device will be used.
 
+- `cloud_init_disk_type` (string) - The type of Cloud-Init disk. Can be `scsi`, `sata`, or `ide`
+  If not given, defaults to `ide`.
```

Comment:
I believe this should update itself based on the comment from `config.go`, have you run `make generate` to update that?